src/popup/components/Header.vue

```diff
@@ -26,7 +26,22 @@
       </Component>
     </div>
 
+    <BtnPlain
```

Comment:
I see little problem with this - this button takes too much space in the header. User can click on empty space after the button which is not intuitive in my opinion.